R/pr.R

```diff
@@ -74,21 +80,35 @@ pr_init <- function(branch) {
 #'   pull request. Default of `NULL` tries to identify the source repo and uses
 #'   the owner of the `upstream` remote, if present, or the owner of `origin`
 #'   otherwise.
+#' @inheritParams git2r_credentials
 #'
 #' @examples
 #' \dontrun{
 #' ## scenario: current project is a local copy of fork of a repo owned by
 #' ## 'tidyverse', not you
 #' pr_fetch(123, owner = "tidyverse")
 #' }
-pr_fetch <- function(number, owner = NULL) {
+pr_fetch <- function(number,
+                     owner = NULL,
+                     credentials = NULL,
+                     auth_token = github_token()) {
```

Comment:
In general, the `pr_*()` functions all gained the args `credentials` and `auth_token`, though I can't imagine actually using them. Anyone who uses these functions heavily will want to have a setup that plays well with default behaviour.

Reply:
I would rather not add these arguments, if the only argument is for consistency. I think we should be moving away from these arguments in general, in favour of better tooling for declaring credentials globally.

Reply:
Yeah, I can see that and what you say is now possible with via `use_git2r_credentials()`. I guess it is also possible for the `auth_token` in a more indirect way, i.e. via the env vars. So we can remove them and make sure error messages / docs tell people what to do if default behaviour doesn't work for them.